app/src/main/java/com/asfoundation/wallet/ui/backup/SystemFileIntentResult.kt

```diff
@@ -0,0 +1,7 @@
+package com.asfoundation.wallet.ui.backup
+
+import androidx.documentfile.provider.DocumentFile
+
+data class SystemFileIntentResult(val documentFile: DocumentFile?) {
```

Comment:
we can set a default value in the primary constructor preventing us from having this secondary.
it seems a little weird.

it would be like: 
`data class SystemFileIntentResult(val documentFile: DocumentFile? = null) `